pip_audit/_cli.py

```diff
@@ -227,10 +236,13 @@ def audit() -> None:
 
         source: DependencySource
         if args.requirements is not None:
+            if args.paths:
+                print("--requirement (-r) and --path arguments cannot be used together")
+                sys.exit(1)
```

Comment:
We can probably use an exclusive argument group for this, to ensure at argument-parse-time that these are never used together: https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group

Reply:
d1ecd3d